modules/apps/dynamic-data-mapping/dynamic-data-mapping-web/src/main/resources/META-INF/resources/js/ddm_form.js

```diff
@@ -3119,14 +3119,14 @@ AUI.add(
 								ddPlugins.push(
 									{
 										cfg: {
-											constrain: '.lfr-form-content'
+											constrain: '.lfr-ddm-container'
```

Comment:
@natocesarrego, Is this class stylized in the forms css?

Reply:
HEy @clovisdasilvaneto I checked and this class is not stylized in the forms css. Thx